engine/src/test/scala/cromwell/jobstore/JobStoreServiceSpec.scala

```diff
@@ -0,0 +1,76 @@
+package cromwell.jobstore
+
+import akka.actor.Props
+import com.typesafe.config.ConfigFactory
+import cromwell.CromwellTestkitSpec
+import cromwell.backend.BackendJobDescriptorKey
+import cromwell.core.{JobOutput, WorkflowId}
+import cromwell.jobstore.JobStoreService._
+import cromwell.jobstore.JobStoreServiceSpec._
+import org.scalatest.Matchers
+import org.specs2.mock.Mockito
+import wdl4s.Call
+import wdl4s.values.WdlString
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+object JobStoreServiceSpec {
+  val MaxWait = 5 seconds
+}
+
+class JobStoreServiceSpec extends CromwellTestkitSpec with Matchers with Mockito {
```

Comment:
Since the `JobStoreService` is just forwarding messages to the `_Reader` and `_Writer`, it's probably not much of a value-add to test it like this?

Especially since the two `_Reader` and `_Writer` classes can be tested with stub databases which don't ever need to reach out to filesystems and/or databases